testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthApplicationWithoutStartupOperationTestCase.java

```diff
@@ -26,10 +26,9 @@
 import org.jboss.dmr.ModelNode;
 
 import java.io.IOException;
+import static org.wildfly.test.integration.microprofile.health.MicroProfileHealthApplicationReadyTestBase.testManagementOperation;
```

Comment:
personally, I think it would be better to move it into a separate class because referencing static method from *Ready* class in *Startup* or *Liveness* might be confusing.

Reply:
Yeah, I have created one class Utility.java. Please suggest if you have any other names.

Reply:
Maybe better would be `MicroProfileHealthUtils`

Reply:
Okay, done.